.gitignore

```diff
@@ -200,4 +200,7 @@ invoice-api/node_modules/
 .DS_Store
 
 #rsa
-rsa
\ No newline at end of file
+rsa
+
+#configuration
+globalConfig.json
```

Comment:
if you are going to want a file for globalConfig, you should at least provide an example version with `globalConfig.json.example`